cmd/src/sbom_fetch.go

```diff
@@ -283,7 +283,19 @@ func extractSBOM(attestationBytes []byte) (string, error) {
 		return "", fmt.Errorf("failed to decode payload: %w", err)
 	}
 
-	return string(decodedPayload), nil
+	// Unmarshal the decoded payload to extract predicate
+	var payload map[string]json.RawMessage
+	if err := json.Unmarshal(decodedPayload, &payload); err != nil {
+		return "", fmt.Errorf("failed to unmarshal decoded payload: %w", err)
+	}
+
+	// Extract just the predicate field
+	predicate, ok := payload["predicate"]
+	if !ok {
+		return "", fmt.Errorf("no predicate field found in payload")
+	}
+
+	return string(predicate), nil
```

Comment:
Just to check: what's in `predicate`? If it's a string, this [will contain the surrounding quotes](https://go.dev/play/), which may not be intentional.

Reply:
That's a good point, I didn't know about that quirk. `predicate` contains a JSON struct (the CycloneDX JSON format) so we're ok in this case.